openmmtools/alchemy.py

```diff
@@ -2038,6 +2060,112 @@ def _alchemically_modify_AmoebaMultipoleForce(self, reference_force, alchemical_
+                only_one_particle_alchemical = ((particle1 in alchemical_region.alchemical_atoms)
+                                             != (particle2 in alchemical_region.alchemical_atoms))
+                if only_one_particle_alchemical:
+                    parameters.append(1)
+                elif both_particles_alchemical:
+                    parameters.append(1 if alchemical_region.annihilate_sterics else 0)
+                else:
+                    parameters.append(0)
+                modified_force.setBondParameters(i, particle1, particle2, parameters)
+
+                # add global parameters
+            modified_force.addGlobalParameter('lambda_sterics', 1.0)
+            modified_force.addGlobalParameter('softcore_alpha', alchemical_region.softcore_alpha)
+            modified_force.addGlobalParameter('softcore_a', alchemical_region.softcore_a)
+            modified_force.addGlobalParameter('softcore_b', alchemical_region.softcore_b)
+            modified_force.addGlobalParameter('softcore_c', alchemical_region.softcore_c)
+            return {'lambda_sterics': [modified_force]}
+        else:
+            return {'': [copy.deepcopy(reference_force)]}
+
+    def _alchemically_modify_CustomNonbondedForce(self, reference_force, alchemical_regions, _):
+
+        energy_function = reference_force.getEnergyFunction()
+
+        # this is a quick fix
+        # TODO: extend to multiple alchemical regions
+        assert len(alchemical_regions) == 1
+        alchemical_region = alchemical_regions[0]
+
+        # Reasons to not modify the force
+        if (len(alchemical_region.alchemical_atoms) == 0
+            # Skip if the force does not depend on radius
+            or not is_variable_in_energy_string(energy_function, "r")
+            or force_has_softcore(energy_function)
+        ):
+            return {'': [copy.deepcopy(reference_force)]}
+
+        # Alchemically modify
+        modified_force = copy.deepcopy(reference_force)
+        if is_nbfix_force(reference_force):
+            softcore_energy_string = modify_vdw_energy_string_softcore(
+                energy_string=energy_function, do_annihilate=alchemical_region.annihilate_sterics
+            )
+            modified_force.setEnergyFunction(softcore_energy_string)
+
+        elif is_custom_lj_force(reference_force):
+            softcore_energy_string = modify_vdw_energy_string_softcore(
+                energy_string=energy_function, do_annihilate=alchemical_region.annihilate_sterics,
+                sigma_string="0.5*(sigma1+sigma2)"
+                # we just recalculate sigma since this is certainly not the bottleneck
+            )
+            modified_force.setEnergyFunction(softcore_energy_string)
+        else:
+            # No fitting modification found
+            return {'': [copy.deepcopy(reference_force)]}
+
+        if is_custom_lj_force(reference_force) or is_nbfix_force(reference_force):
```

Comment:
I think this if-clause is always `True`, so you could remove one level of indentation here.